charts/metering-ansible-operator/values.yaml

```diff
@@ -36,6 +36,13 @@ operator:
       verbs:
       - get
       - list
+    # grants access to view the networking config
```

Comment:
oof, so many places to add the exact same lines.

Reply:
I know, we manage too many sets of manifest files.